app/controllers/ErrorsController.scala

```diff
@@ -26,13 +26,14 @@ import uk.gov.hmrc.play.frontend.controller.FrontendController
 import scala.concurrent.Future
 
 @Singleton
-class ErrorsController @Inject()(val appConfig: AppConfig, val messagesApi: MessagesApi) extends FrontendController with I18nSupport {
+class ErrorsController @Inject()(val messagesApi: MessagesApi, implicit val appConfig: AppConfig)
+  extends FrontendController with I18nSupport {
 
   val sessionTimeout: Action[AnyContent] = Action.async { implicit request =>
-    Future.successful(Ok(views.html.errors.sessionTimeout(appConfig)))
+    Future.successful(Ok(views.html.errors.sessionTimeout()))
```

Comment:
You can also remove the `Future.successful(...)` wrapper if you remove `.async`. Play will take care of enclosing the `Ok(..)` result in a Future (more here https://www.playframework.com/documentation/2.5.x/ScalaAsync)